packages/@aws-cdk/aws-lambda/lib/function-hash.ts

```diff
@@ -1,10 +1,10 @@
-import { CfnResource, FeatureFlags, Stack } from '@aws-cdk/core';
+import { CfnResource, FeatureFlags, Stack, Token } from '@aws-cdk/core';
 import { md5hash } from '@aws-cdk/core/lib/helpers-internal';
 import { LAMBDA_RECOGNIZE_LAYER_VERSION, LAMBDA_RECOGNIZE_VERSION_PROPS } from '@aws-cdk/cx-api';
 import { Function as LambdaFunction } from './function';
 import { ILayerVersion } from './layers';
 
-export function calculateFunctionHash(fn: LambdaFunction) {
+export function calculateFunctionHash(fn: LambdaFunction, additional: string = '') {
```

Comment:
[nit] is the `additional` more like `salt` or would that name be too obtruse for the intent?

Reply:
It is actually `salt`. I suppose I could have included that word here, but in the public API I purposely didn't want to use the word `salt` because that's extremely computer-sciency and I was trying to sound approachable.